### ISSUE TITLE
Robust Accuracy results not matching

### ISSUE THREAD

**Issue by code-Assasin** [dicarlolab/vonenet#21]
Firstly, thank you for open sourcing the code for your paper. It has been really helpful !! 

I had a small query regarding the robust evaluation of models.
I tried to evaluate the pretrained VoneResNet50 model with standard PGD with EOT and I get the following results: 

```
robust accuracy (top1):0.3666
robust accuracy (top5):0.635
```
My PGD parameters were as follows :
```
iterations : 64
norm : L inifity
epsilon: 0.0009803921569 (= 1/1020)
eot_iterations : 8
Library: advertorch 
```
I used the code in this [PR](https://github.com/dicarlolab/vonenet/pull/12) and also checked with [another library](https://github.com/Harry24k/adversarial-attacks-pytorch) 

It seems like the top-5 accuracy is closer to the accuracy mentioned in the paper. I'm confused since the paper mentions that the accuracy is always top-1?

**Comment by tiagogmarques** [comment]
Those results seem to match the VOneResNet50 without stochasticity. Are you sure you are using the correct model? Thank you

**Comment by tiagogmarques** [comment]
Could you please let us know exactly the code that you ran so that we try to replicate the issue?

**Comment by code-Assasin** [comment]
> Those results seem to match the VOneResNet50 without stochasticity. Are you sure you are using the correct model?Thank you

It was my mistake, sorry. As you correctly pointed out I removed the stochasticity for evaluating the robustness due to some gradient errors I had:   
```
model.module.vone_block.simple = nn.ReLU(inplace=False)
model.module.vone_block.noise = nn.ReLU(inplace=False)
```
I commented out the second line and now it works fine. 
Sorry for the trouble and thanks for your help :)
